build/lammps.py

```diff
@@ -84,33 +90,29 @@ def native(ctx, clean=False):
 
     res = run(cmake_str, shell=True, cwd=work_dir, env=env_vars)
     if res.returncode != 0:
-        raise RuntimeError("LAMMPS native CMake config failed")
+        raise RuntimeError("LAMMPS CMake config failed")
 
     res = run("ninja", shell=True, cwd=work_dir)
     if res.returncode != 0:
-        raise RuntimeError("LAMMPS native build failed")
+        raise RuntimeError("LAMMPS build failed")
 
     res = run("ninja install", shell=True, cwd=work_dir)
     if res.returncode != 0:
         raise RuntimeError("LAMMPS install failed")
 
 
-@task
-def copy_wasm(ctx, clean=False):
-    """
-    Manually copy the LAMMPS binary to the Faasm func dir
-    """
-    install_dir = join(LAMMPS_DIR, "install", "bin")
-    faasm_func_dir = "/usr/local/code/faasm/wasm/lammps/main"
-    cmd = [
-        "cp",
-        "{}/lmp".format(install_dir),
-        "{}/function.wasm".format(faasm_func_dir),
-    ]
-
-    cmd = " ".join(cmd)
-    print(cmd)
-
-    res = run(cmd, shell=True)
-    if res.returncode != 0:
-        raise RuntimeError("Copying wasm file failed")
+if __name__ == "__main__":
+    if len(sys.argv) < 2:
+        print("usage: ./build/lammps.py native|faasm [--clean]")
+        sys.exit(1)
+    elif sys.argv[1] not in ["native", "faasm"]:
+        print("usage: ./build/lammps.py native|faasm [--clean]")
+        sys.exit(1)
+    else:
+        clean = False
+        if len(sys.argv) > 2 and sys.argv[2] == "--clean":
+            clean = True
+        if sys.argv[1] == "native":
+            build_native(clean)
+        else:
+            build_faasm(clean)
```

Comment:
Could be done more cleanly with [argparse](https://docs.python.org/3/library/argparse.html), but fine for now